gitops/snapshot.go

```diff
@@ -703,3 +706,25 @@ func RemoveIntegrationTestRerunLabel(adapterClient client.Client, ctx context.Co
 
 	return nil
 }
+
+func GetLatestUpdateTime(snapshot *applicationapiv1alpha1.Snapshot) (time.Time, error) {
```

Comment:
While we do exercise these functions somewhat in the status/reporters unit tests, I think we'll also need some dedicated unit tests for them to have enough code coverage.

Reply:
Added for two cases within existing unit tests, would this coverage suffice?

Reply:
I was thinking more along the lines of adding unit tests to the [gitops/snapshot_test.go](https://github.com/redhat-appstudio/integration-service/blob/main/gitops/snapshot_test.go) to satisfy codecov.